app/models/piece.rb

```diff
@@ -14,7 +14,7 @@ def move_to!(params)
       # Step 2: Check to see if the piece is of the oppposite color.
       if color != enemy_piece[:color]
         # Step 3: Remove piece from the board.
-        enemy_piece.update_attributes(horizontal_position: nil, vertical_position: nil)
+        enemy_piece.destroy
```

Comment:
Really? That's strange...I pulled the code in and tested it right now. I'm seeing an indented line 6. If I place the cursor at line 6, it doesn't line up with the cursor at line 9 (which is another empty line). Deleting the indentation fixed the trailing whitespace error for me. If it still doesn't work, I'll fix it in my next pull request.

I guess if it's still broken for you, someone else can fix it in their next PR.